src/content/manifest/environment-variables.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Environment variables
+description: Use Okteto's built-in environment variables to streamline your Okteto manifest.
+sidebar_label: Environment variables
+id: environment-variables
+---
+
+The Okteto manifest utilizes environment variables with each section supporting specific ENV VARs. Additional ENV VARs are added regularly and it's important to understand the existing ones and their usage. Currently, there are three types of ENV VARs to be aware of:
+
+## Using environment variables
```

Comment:
I wouldn't start here, `OKTETO_ENV` is the most advanced scenario, and not needed in 99% of the cases.
I would start explaining user defined variables and its precedence. You can have (in order of preference):
- Local variables
- Deploy time variables
- Catalog variables
- User variables (in settings)
- Admin variables

Reply:
@pchico83 I'll make this edit. For my clarity: this list is the order, from top to bottom, that we recommend implementing/using variables?

Does this list also represent, in reverse order, when variables override the ones below? For example, a user variable defined in `settings` would override a locally-defined or deploy time variable?